docs/docs/50-running/02-networking.md

```diff
@@ -120,3 +120,49 @@ To highlight the level of control this gives you, consider the following:
 If you set your cluster domain to `my-company.com`, deployed into the default `acorn` namespace, named your app `blog`, and the container name in the Acornfile was `default`, the published endpoint for your app would be: `http://blog.my-company.com`
 
 In addition to this method of controlling your endpoint, you can publish to an explicit external name using the `--publish` flag. See the [publishing ports](#publish-individual-ports) section for more details.
+## Routers
+If you have two containers normally they would be exposed as two different HTTP services. The router feature allows you to expose these two containers as a single HTTP service with separate routes. Let's take a look at how we can achieve this below.
```

Comment:
What are your thoughts on this? Personally I think that it makes it clearer that this scales to more than two containers which may be non-obvious for beginners.

```suggestion
If you have multiple containers normally they would be exposed as two different HTTP services. The router feature allows you to expose those containers as a single HTTP service with separate routes. Let's take a look at how we can achieve this with two containers below.
```

Reply:
I like this better